src/main/java/org/jenkinsci/test/acceptance/docker/DockerFixture.java

```diff
@@ -38,4 +39,23 @@
      * Ip address to bind to
      */
     String bindIp() default "127.0.0.1";
-}
+
+    /**
+     * Path of Dockerfile file.
+     *
+     * <p>
+     *  By default Dockerfile fixture should be placed in the resource directory
+     *  and at the same package as DockerContainer subtype.
+     * </p>
+     * <p>
+     *  If this attribute is present, Dockerfile file specified in the attribute
+     *  is used as Dockerfile fixture.
```

Comment:
According to the documentation and the tests, this attribute specifies the (resource) path to the _folder_ containing `Dockerfile`, not the `Dockerfile` itself. So this Javadoc is misleading, as is the name of the attribute.